acceptance/tests/apply_plan_ssh.rb

```diff
@@ -160,15 +169,20 @@
       end
 
       ssh_nodes.each do |node|
-        # Verify that node succeeded
         host = node.hostname
-        result = json.select { |n| n['node'] == host }
-        assert_equal('failure', result[0]['status'],
-                     "The task did not fail on #{host}")
-        assert_match(/Permission denied/, result[0]['result']['_error']['msg'])
-
-        # Verify that files were not created on the target
-        on(node, "cat #{restricted_filepath}/hello.txt", acceptable_exit_codes: [1])
+        result = json.select { |n| n['node'] == host }.first
+        assert_equal('success', result['status'],
+                     "The task failed on #{host}")
+
+        stat = if node['platform'] =~ /osx/
+                 "stat -f \"%Su\" #{filepath}/hello.txt"
```

Comment:
Are these escaped quotes necessary here? It looks like, when we interpolate this inside of more quotes, this just results in `%Su` being the only _unquoted_ part of the command.

Reply:
Oo good catch